### PR TITLE
fix: buildURL error

### LINKED ISSUE #99: Missing "./lib/helpers/buildURL" export in "axios" package
Hello, I have encountered the following error.

X [ERROR] [plugin vite:dep-pre-bundle] Missing "./lib/helpers/buildURL" export in "axios" package

    node_modules/axios-extensions/esm/utils/buildSortedURL.js:7:21:
      7 │ import buildURL from 'axios/lib/helpers/buildURL';

https://github.com/kuitos/axios-extensions/blob/0b64fd7b155efe0fa65d1025da3ed04d65faef5f/src/utils/buildSortedURL.ts#L8

    "vite": "^4.0.0"
    
    "eslint": "^8.22.0",
    
    "axios": "^1.2.1",
    "axios-cancel": "^0.2.2",
    "axios-extensions": "^3.1.6",

### PR DESCRIPTION
**Pull request by mengqiuleo**
fix error: Missing "./lib/helpers/buildURL" export in "axios" package

* resolves #99

### CODE REVIEW

**Comment by mengqiuleo** [comment]
#99

**Comment by mengqiuleo** [comment]
with axios 1.x, we can use axios.getUri API, https://github.com/Gerhut/axios-debug-log/commit/df0f23a8c2258c4764227ea93dcd7951a651d90a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

**Comment by mengqiuleo** [comment]
When using the axios 1.x version, typescript error for `AxiosRequestConfig`, https://github.com/axios/axios/issues/5573

**Comment by mengqiuleo** [comment]
All test cases have passed successfully.

File: yarn.lock
Comment (kuitos) [inline yarn.lock:1]: yarn.lock does't need to be submitted

**Review by kuitos — CHANGES_REQUESTED** [review]
Nice work~
But `axios.getUri` depends on axios 1.x, it's breaking change for who using axios 0.x, maybe we should publish a major version like 4.0 for axiso-extensions?

**Comment by mengqiuleo** [comment]
> Nice work~ But `axios.getUri` depends on axios 1.x, it's breaking change for who using axios 0.x, maybe we should publish a major version like 4.0 for axiso-extensions?

@kuitos, I've tested `axios.getUri` locally, it's **compatible with axios 0.x**, and I removed the code I originally added for ts type，it's not compatible，use `@ ts-ignore`  to resolve ts type error.

**Comment by kuitos** [comment]
Thanks for your nice work~

**Comment by onlywei** [comment]
Will this ever be published to npm?

**Comment by botmaster** [comment]
Please, publish this fix to npm 😔

**Comment by mpicardvolvo** [comment]
This project has been abandoned, too bad

**Pull request merged**